[KB-246] Z-score Normalization (standard scaler) (#153)
chhoumann/thesis-chemcam: report_thesis/src/sections/background.tex

\section{Background}\label{sec:background}
\subsection{Preprocessing}

\subsubsection{Z-score Normalization}

> Z-score normalization, also known as standard score normalization, transforms the data into a distribution with a mean of zero and a standard deviation of one. 

Reviewer: ```suggestion
Z-score normalization, also known as standard score normalization, transforms data to have a mean of zero and a standard deviation of one.
```

> This normalization is useful when the actual minimum and maximum of a feature are unknown or when there are outliers that may skew the distribution significantly. 

Reviewer: ```suggestion
This technique is useful when the actual minimum and maximum of a feature are unknown or when outliers may significantly skew the distribution.
```

The formula for Z-score normalization is given by:

> $$ v' = \frac{v - \overline{F}}{\sigma_F} $$

Reviewer: ```suggestion
$$ 
v' = \frac{v - \overline{F}}{\sigma_F},
$$
```

where $v$ is the original value, $\overline{F}$ is the mean of the feature $F$, and $\sigma_F$ is the standard deviation of $F$. 

> By transforming the data using the Z-score, each value reflects the number of standard deviations away from the mean. 

Reviewer: ```suggestion
By transforming the data using the Z-score, each value reflects its distance from the mean in terms of standard deviations.
```

> This method is helpful in environments where the data needs to be standardized to fit a normal distribution, making it easier to compare differences in data points~\cite{dataminingConcepts}.

Reviewer: ```suggestion
Z-score normalization is particularly advantageous in scenarios where data features have different units or scales, or when preparing data for algorithms that assume normally distributed inputs~\cite{dataminingConcepts}.
```

\section{Background}\label{sec:background}
\subsection{Preprocessing}

\subsubsection{Z-score Normalization}

> Z-score normalization, also known as standard score normalization, transforms data to have a mean of zero and a standard deviation of one.

Reviewer 2: Known as standardization, no? Never heard of `standard score normalization`.
